src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -215,7 +218,7 @@ function FloatingActionButtonAndPopover(props) {
                     ...(!props.isLoading && !Policy.hasActiveFreePolicy(props.allPolicies)
                         ? [
                               {
-                                  displayInDefaultIconColor: true,
+                                  displayInDefaultIconColor: isNative,
```

Comment:
I don't think this is stated in your proposal and using Platform specific code like this one is not something encouraged.

Reply:
@getusha Yes, I had to add this because without it native platform will simply apply a `tintColor` over the entire icon which hides the icon shape.

https://github.com/Expensify/App/blob/dc3e19bdcc95023d74cd7fdd433f6208707eaaf8/src/components/ImageSVG/index.native.tsx#L7-L15

This kind of check is already used in Share Code page. Otherwise we'll have to duplicate the entire logic into a platform specific file with `.native.js` extension.

https://github.com/Expensify/App/blob/dc3e19bdcc95023d74cd7fdd433f6208707eaaf8/src/pages/ShareCodePage.js#L78

Reply:
Lets create a utility function that'll return the boolean value `getDisplayInDefaultIconColor` that'll return false for native.

Reply:
This condition is quite specific for NewWorkspace icon. We wouldn't want it to be used across the app. Naming it `getDisplayInDefaultIconColor` would be quite confusing.

Reply:
we can name it `getShouldDisplayInDefaultIconColor` or any other name that'll suit the function.

>Put all platform specific code in dedicated files and folders, like /platform, and reject any PR that attempts to put platform-specific code anywhere else. This maintains a strict separation between business logic and platform code.

Reply:
Done.